src/pulselistener/pulselistener/code_review.py

```diff
@@ -57,6 +57,8 @@ def get_repositories(self, repositories, cache_root):
 
             # Receive build from webserver
             build = await self.bus.receive(QUEUE_WEB_BUILDS)
+            if build is None:
```

Comment:
When does this happen? Do we risk entering in an infinite loop?

Reply:
This happen when i do `rpush webserver:out "THIS IS A CRASH"` :smile: 

When i was testing the integration, it was immediately crashing the whole program, so i prefer to have a solution for bad data...

Reply:
Maybe we should assert instead? If we have a stream of bad data we will loop forever otherwise

Reply:
Done.